contrib/tools/meta.md/cterms.md

```diff
@@ -203,15 +203,15 @@ Right now we just take *any* maximal sort in `C1 /\ C2`, but we really need to t
 ```maude
     op joint-sort : Sort ModuleDeclSet ModuleDeclSet -> Sort .
     ----------------------------------------------------------
-    ceq joint-sort(S, MDS, MDS') = S if S inS intersect(MDS, MDS') .
-    ceq joint-sort(S, MDS, MDS') = joint-sort(S, MDS', MDS) if (not S inS MDS) /\ S inS MDS .
+    ceq joint-sort(S, MDS, MDS') = S if S inS moduleIntersect(MDS, MDS') .
+    ceq joint-sort(S, MDS, MDS') = joint-sort(S, MDS', MDS) if (not S inS MDS) /\ S inS MDS' .
     ceq joint-sort(S, MDS, MDS') = if S inS MDS'' then S else #top-sort(MDS'') fi
-                                if S inS MDS /\ MDS'' := intersect(connected-component(MDS, (sorts S .)), MDS') .
+                                if S inS MDS /\ MDS'' := moduleIntersect(connected-component(MDS, (sorts S .)), MDS') .
 
     op joint-sort : Sort Module           Module           -> Sort .
     op joint-sort : Sort ModuleExpression ModuleExpression -> Sort .
     ----------------------------------------------------------------
-    eq joint-sort(S, ME,  ME')  = joint-sort(S, intersect(ME, ME'), intersect(ME, ME')) .
+    eq joint-sort(S, ME,  ME')  = joint-sort(S, moduleIntersect(ME, ME'), moduleIntersect(ME, ME')) .
     eq joint-sort(S, MOD, MOD') = joint-sort(S, asTemplate(MOD), asTemplate(MOD')) .
 
```

Comment:
This looks strange to me, why would `joint-sort` effectively pass the same module as both arguments here? It calls `moduleIntersect(ME, ME')` in both positions. This bug goes back to when the code was introduced, I think it should be `joint-sort(S, upModule(ME, true), upModule(ME', true))` instead.

Reply:
Good catch. I'll cook up a test case before fixing it.